src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -1,5 +1,10 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+import javax.persistence.PrimaryKeyJoinColumn;
+
+@Entity
+@PrimaryKeyJoinColumn(name = "coach_id")
```

Comment:
what does this annotation do?

Reply:
saw it in guide, now understand that we can go without it
fixed